tests/e2e/server/index.js

```diff
@@ -0,0 +1,40 @@
+import { Server, Response } from 'miragejs';
+import { accessToken, initial, second, userInfo } from './data';
+
+new Server({
+  routes() {
+    this.urlPrefix = 'https://forgerock-sdk-samples.com:3000';
+    this.namespace = '/am';
+
+    this.post('/json/realms/root/authenticate', (schema, request) => {
+      if (!request.requestBody) {
+        return initial;
+      } else {
+        return second;
+      }
+    });
+
+    this.get('/oauth2/realms/root/authorize', (schema, request) => {
+      return new Response(302, {
+        location:
+          'https://forgerock-sdk-samples.com:3000/callback?code=OE1TTr66di0N12cN3LbvZw15if4&iss=https://openam-devjustin3.forgeblocks.com/am/oauth2&state=157593870032300&client_id=d923852a41c7d4d90a804df7e3357092',
```

Comment:
I'm starting to feel like these base urls should be defined as constants somewhere. =)

Reply:
Will do :)